pkg/controller/llmisvc/DEV.md

```diff
@@ -53,16 +59,52 @@ curl http://${LB_IP}/${NS}/${LLM_ISVC_NAME}/v1/completions  \
 You should see:
 
 ```shell
-curl http://${LB_IP}/${NS}/${LLM_ISVC_NAME}/v1/completions  \
+{
+  "id": "cmpl-f0601f1b-66cc-4f0c-bd0c-cc93c8afd9ec",
+  "object": "text_completion",
+  "created": 1751477229,
+  "model": "facebook/opt-125m",
+  "choices": [
+    {
+      "index": 0,
+      "text": " big place and I'd imagine it will stay that way. Until the US rel",
+      "logprobs": null,
+      "finish_reason": "length",
+      "stop_reason": null,
+      "prompt_logprobs": null
+    }
+  ],
+  "usage": {
+    "prompt_tokens": 5,
+    "total_tokens": 21,
+    "completion_tokens": 16,
+    "prompt_tokens_details": null
+  },
+  "kv_transfer_params": null
+}
+```
+
+Or by using populated URLs from LLMInferenceService status:
+
+```shell
+curl $(kubectl get llmisvc -n $NS -o=jsonpath='{.items[0].status.addresses[0].url}')/v1/completions  \
```

Comment:
maybe for general use we could simply use `.items[0].status.url`

Reply:
See [note](https://github.com/opendatahub-io/kserve/pull/719/files/efe1d9d5dbd1879a356f9a42cd5406c52f74c7e1#diff-68aba1e6bdef358fb9a3b952bf0f8f8e195bf48dd663155af4178f18f4926a0bR98) below

Reply:
but for general use, yes, assuming it's externally exposed